### PR TITLE
Scenario end: solo campaign

### PR DESCRIPTION
**Pull request by DarthGandalf**
both-5
both-39

### CODE REVIEW

File: sections/scenario_end.tex [hunk 1]
```diff
@@ -21,8 +21,17 @@
...
+  \item all the \hyperlink{Statistic}{Statistic} cards from your deck,
+  \item the level 1 \hyperlink{Specialty}{Specialty} card,
+  \item 5 other non-Specialty cards of your choice from your deck.
+\end{itemize}
+Skip steps 11-13 of Setup for the next scenario of the campaign.
+
 \vspace*{\fill}
 \hspace{-6em}
-\includegraphics[width=2\linewidth]{\art/medusa.png}
+\includegraphics[width=1.5\linewidth]{\art/medusa.png}
```
Comment (qwrtln) [inline sections/scenario_end.tex:35]: Small Medusa doesn't look good. Move some of the text to the left column. If Medusa's tail will be too close to the text, move it slightly to the right by modifying `\hspace` value.

File: sections/game_modes.tex [hunk 2]
```diff
@@ -11,7 +11,7 @@ \subsection*{Clash}
 
 \subsection*{Campaign}
 A single player mode of interconnected Scenarios against an enemy AI.
-For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules}.
+For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules} and \hyperlink{End}{Scenario End}.
```
Comment (qwrtln) [inline sections/game_modes.tex:14]:
```suggestion
For rules unique to Solo Mode\index{Solo Mode}, see \hyperlink{AIrules}{the AI rules} and \hyperlink{End}{Scenario end}.
```
It makes sense to capitalise everything in the titles, but I think "end" can be lowercase.

File: sections/scenario_end.tex (hunk 1 above)
Comment (DarthGandalf) [reply sections/scenario_end.tex]: What about now? If I don't resize it at all, it gets pushed to the next page, which is much worse.

File: sections/scenario_end.tex [hunk 3]
```diff
@@ -16,13 +16,22 @@
...
+  \item all the \hyperlink{Statistic}{Statistic} cards from your deck,
+  \item the level 1 \hyperlink{Specialty}{Specialty} card,
+  \item 5 other non-Specialty cards of your choice from your deck.
+\end{itemize}
+Skip steps 11-13 of Setup for the next scenario of the campaign.
+
 \vspace*{\fill}
 \hspace{-6em}
-\includegraphics[width=2\linewidth]{\art/medusa.png}
+\includegraphics[width=1.8\linewidth]{\art/medusa.png}
```
Comment (qwrtln) [inline sections/scenario_end.tex:35]: image

Comment (qwrtln) [reply sections/scenario_end.tex:35]: Let me push an updated Medusa and see how it works. Hang on.

Comment (qwrtln) [reply sections/scenario_end.tex:35]: There. Now restore `2` and see how it looks.

Comment (DarthGandalf) [reply sections/scenario_end.tex:35]: I don't know what you changed, but please build it, and see how it doesn't work

Comment (DarthGandalf) [reply sections/scenario_end.tex]: PTAL

Comment (qwrtln) [reply sections/scenario_end.tex]: A little furgher to the right

Comment (DarthGandalf) [reply sections/scenario_end.tex]: moved

**Review by Heegu-sama — APPROVED** [review]

**Review by qwrtln — APPROVED** [review]

**Pull request merged**